quickstart/201-azfw-with-secure-hub/variables.tf

```diff
@@ -0,0 +1,30 @@
+// Create Variables for Location and Tags
```

Comment:
Each variable should contain all of the following attributes:

* type
* description
* default

Reply:
added attributes